### PR TITLE
AlienVault OTX v2 - refactor reputation commands and relations

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/35557
related: https://github.com/demisto/etc/issues/35284

## Description
- [x] refactor the commands to use command_results
add relationship to it + tests:
- [x] url
Screen Shot 2021-04-26 at 13 38 43
- [x] ip 
Screen Shot 2021-04-26 at 16 37 00
- [x] file - see comments in the related issue.
- [x] domain - see comments in the related issue.

## Minimum version of Cortex XSOAR
- [x] 5.0.0 - refactor
- [x] 6.2.0 - relationships

## Does it break backward compatibility?
   - [X] No - tested with UT in the commit `c0d647173800c330a1866af9549a820d601e97a7`

## Must have
- [x] Tests
- [x] Documentation - ~~eventually will re create the entire readme~~

### CODE REVIEW

**Comment by bakatzir** [comment]
Self reminder to re create the README.

File: Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py
Comment (yuvalbenshalom) [inline Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py:166]: I think that this is blocked by `Attack Pattern` indicator as this indicator dose not exist yet.
what we have at the moment is `STIX Attack Pattern`.

**Review by yuvalbenshalom — COMMENTED** [review]
looks good but I think we need to wait for `Attack Pattern` indicator type.

File: Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py
Comment (bakatzir) [reply Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py:166]: correct. we map by the details fields which should indeed be `STIX Attack Pattern`.
https://github.com/demisto/content/blob/master/Packs/CommonTypes/IndicatorTypes/reputation-STIX_Attack_Pattern.json#L5

I can wither change to `STIX Attack Pattern` or wait. your call.

Comment (yuvalbenshalom) [reply Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py]: yes, please change to `STIX Attack Pattern`

**Review by yuvalbenshalom — APPROVED** [review]

**Review by orhovy — APPROVED** [review]

**Pull request merged**
